src/panel/mixins/translation.js

```diff
@@ -16,6 +16,36 @@ export default {
           if (!Object.keys(translatableBlocks).includes(block.type)) continue;
 
           for (const blockFieldKey of Object.keys(block.content)) {
+
+            // Handle structures
+            if (
+              isObject(translatableBlocks[block.type]) &&
```

Comment:
Currently "isObject" also returns true for arrays. I don't know if this is intended. But here i want to specifically check for objects only.

Reply:
You can use `isObject(obj) && !Array.isArray(obj)`.

Reply:
Yeah that's what I ended up doing. Just thought I'd make you aware of this potential pitfall. But it seems like you already are aware of it :)